setup.py

```diff
@@ -63,7 +63,7 @@
 REQUIRED_PKGS = [
     "torch>=1.4.0",
     "transformers>=4.31.0",
-    "numpy>=1.18.2,<2.0.0",
+    "numpy>=1.18.2",
```

Comment:
To be safe, let's add the upper bound for the next major release:
```suggestion
    "numpy>=1.18.2,<3.0.0",
```

Reply:
Are you sure you really want this? There hasn't been any any known issues with this version yet. By this logic we should add upper bound pins on all the other deps too?

I'll make the change if you really want it. Just want to make sure you do first.

Reply:
I took a look at our other libs and indeed we don't upper bound `numpy`, so let's keep it as is. Thanks!